tests/lib/rules/no-arrow-function-computed-properties.js

```diff
@@ -0,0 +1,46 @@
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/no-arrow-function-computed-properties');
+const RuleTester = require('eslint').RuleTester;
+
+const { ERROR_MESSAGE } = rule;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+  },
+});
+ruleTester.run('no-arrow-function-computed-properties', rule, {
+  valid: [
+    'computed()',
+    'computed(function() { return 123; })',
+    "computed('prop', function() { return this.prop; })",
+    "computed('prop', function() { return this.prop; }).volatile()",
+    "computed.map('products', product => { return someFunction(product); })",
```

Comment:
I don't particularly think we should allow this, any specific reason you added it to valid tests?

Reply:
I originally included this here because arrow functions seem slightly more acceptable with `computed.map` or `computed.sort` where it's less likely that the computed property will use another property on the same object via `this`.

However, in response to your comment, I updated the rule, and moved this test case to the `invalid` section, assuming you still prefer that!